lib/mix_audit/formatting/human.ex

```diff
@@ -23,6 +23,7 @@ defmodule MixAudit.Formatting.Human do
     #{colorized_text("Title:", :red)} #{String.trim(vulnerability.advisory.title)}
     #{colorized_text("Vulnerable versions:", :red)} #{versions(vulnerability.advisory.vulnerable_version_ranges)}
     #{colorized_text("First patched versions:", :red)} #{versions(vulnerability.advisory.first_patched_versions)}
+    #{colorized_text("Severity:", :red)} #{vulnerability.advisory.severity}
```

Comment:
I think we should keep the version information as lowest as possible.

```suggestion
    #{colorized_text("Severity:", :red)} #{vulnerability.advisory.severity}
    #{colorized_text("Vulnerable versions:", :red)} #{versions(vulnerability.advisory.vulnerable_version_ranges)}
    #{colorized_text("First patched versions:", :red)} #{versions(vulnerability.advisory.first_patched_versions)}
```